Products/ZenEvents/EventClass.py

```diff
@@ -67,52 +71,56 @@ class EventClass(EventClassPropertyMixin, Organizer, ManagedEntity, ZenPackable)
+            "instances",
+            ToManyCont(
+                ToOne,
+                "Products.ZenEvents.EventClassInst",
+                "eventClass"
+            )
+        ),
+    )
 
+    _properties = Organizer._properties + EventClassPropertyMixin._properties
 
     # Screen action bindings (and tab definitions)
     factory_type_information = (
         {
-            'id'             : 'EventClass',
-            'meta_type'      : 'EventClass',
-            'description'    : """Base class for all event classes""",
-            'icon'           : 'EventClass.gif',
-            'product'        : 'ZenEvents',
-            'factory'        : 'manage_addEventClass',
-            'immediate_view' : 'eventClassStatus',
-            'actions'        :
-            (
-                { 'id'            : 'classes'
-                , 'name'          : 'Classes'
-                , 'action'        : 'eventClassStatus'
-                , 'permissions'   : (
-                  Permissions.view,)
+            'id': 'EventClass',
+            'meta_type': 'EventClass',
+            'description': """Base class for all event classes""",
+            'icon': 'EventClass.gif',
+            'product': 'ZenEvents',
+            'factory': 'manage_addEventClass',
+            'immediate_view': 'eventClassStatus',
+            'actions': (
+                {
+                    'id': 'classes',
+                    'name': 'Classes',
+                    'action': 'eventClassStatus',
+                    'permissions': (Permissions.view,)
                 },
-                { 'id'            : 'eventList'
-                , 'name'          : 'Mappings'
-                , 'action'        : 'eventMappingList'
-                , 'permissions'   : (
-                  Permissions.view,)
+                {
+                    'id': 'eventList',
+                    'name': 'Mappings',
+                    'action': 'eventMappingList',
+                    'permissions': (Permissions.view,)
                 },
-                { 'id'            : 'events'
-                , 'name'          : 'Events'
-                , 'action'        : 'viewEvents'
-                , 'permissions'   : (
-                  Permissions.view,)
+                {
+                    'id': 'events',
+                    'name': 'Events',
+                    'action': 'viewEvents',
+                    'permissions': (Permissions.view,)
                 },
-                { 'id'            : 'config'
-                , 'name'          : 'Configuration Properties'
-                , 'action'        : 'zPropertyEditNew'
-                , 'permissions'   : ("Change Device",)
+                {
+                    'id': 'config',
+                    'name': 'Configuration Properties',
+                    'action': 'zPropertyEditNew',
+                    'permissions': ("Change Device",)
                 },
             )
-         },
-        )
+        },
+    )
```

Comment:
I've found that the conventional Javascript object literal style is more compact and works in Python as well (and still appears to be PEP8 compliant), e.g.
```
    some_variable = ({
        "id":    "id01",
        "value": "value01"
    }, {
        "id":    "id02",
        "value": "value02"
    },)
```
instead of
```
    some_variable = (
        {
            "id":    "id01",
            "value": "value01"
        },
        {
            "id":    "id02",
            "value": "value02"
        },
    )
```
This is just a suggestion.  If it looks too weird, you can ignore it.

Reply:
It looks fine for me :+1: